moodlecli/moodle.py

```diff
@@ -223,3 +225,6 @@ def get_users_by_course(self, course_id):
 
     def get_course_enrolment_url(self, course_id):
         return f"{self.moodle_url}/enrol/index.php?id={course_id}"
+
+    def get_user_uuids(self, user_ids):
+        return self._get(MOODLE_FUNC_GET_USER_UUIDS, user_ids)
```

Comment:
This looks incorrect to me (assuming `user_ids` is intended to be a list of ints). You have to create the intended data structure that Moodle will expect. I'm fairly certain this will throw an error if `user_ids` is not `[]`.